### ISSUE TITLE
can you help me with this?

### ISSUE THREAD

**Issue by Dendakoduong** [TristanBonnal/Ecommerce-Symfony#3]
hi, thanks for the project, it work perfect but when I try the Admin account, it has the issue 
_When using date/time fields in EasyAdmin backends, you must install and enable the PHP Intl extension, which is used to format date/time values._

I have delete the semi colon in XAMPP, try to install using command composer require symfony/intl but still have this message. Any idea? Thanks for reading

**Comment by TristanBonnal** [comment]
Hello,  you need to install the php-intl module if you haven't already done so.
To check if it's already installed, type the command on linux : 
```bash
php -m
```
If it's not in the list, type the command :
```bash
sudo apt install php8.1-intl
```
Replace 8.1 with your version of php.

**Comment by Dendakoduong** [comment]
Hi, thanks for spending time to reply to me. But I use Windows. I have tried the solution at [here](https://github.com/symfony/symfony/discussions/45613). First, it won't work. Somehow, magically, when I restarted the computer, it worked. I am very honored to receive a reply from the author, by the way. Your project is beneficial for me in learning and understanding how the Symfony framework works. Once again, merci beaucoup, and have a nice day, sir.
